### ISSUE TITLE
rcqtrainer parameter setting problem

### ISSUE THREAD

**Issue by captainzhu123** [armandli/ReconChessRL#1]
hello。I would like to ask the following questions about parameter settings during the entire training process. The total training step length is 2620. How do you get this parameter? And why the sense and action use different loss functions, and how the specific loss functions are defined  .
